sources/install/package_c2.sh

```diff
@@ -15,21 +15,16 @@ function install_pwncat() {
 }
 
 function install_metasploit() {
-    # CODE-CHECK-WHITELIST=add-aliases
+    # CODE-CHECK-WHITELIST=add-history
```

Comment:
This needs to be replaced with `# CODE-CHECK-WHITELIST=add-history,add-aliases` to pass the code compliance checks

Reply:
there are aliases for metasploit

Reply:
my bad, didn't see that